src/Concerns/Testing.php

```diff
@@ -33,7 +33,7 @@ trait Testing
     /**
      * The Illuminate application instance.
      *
-     * @var \Illuminate\Foundation\Application
+     * @var \Illuminate\Foundation\Application|null
```

Comment:
Extreme breaking change... Please announce it

Reply:
This is only for testing packages, and shouldn't affect `app()`.

Reply:
this has triggered our ci systems on many packages, with phpstan flagging that `$this->app` may be null, but i'm not sure i understand the case when it can be null?

to get around this, we're having to add `$this->assertNotNull($this->app)` in our test cases which is a bit strange to have to assert that the test framework is doing what it is supposed to?

is the actual preferred fix to use the `app()` helper? if so, why does `$this->app` even exist?

Reply:
![image](https://user-images.githubusercontent.com/172966/201902869-326bd0a7-7d1f-4842-b205-97efb6354212.png)

`$this->app` will be `null` before `setUp()` and after `tearDown()` and phpstan isn't smart enough in this case.

Reply:
so as a consumer of testbench, what do you suggest we do to avoid these false positives?

Reply:
As a consumer you should run analyse on your code, not tests

Reply:
not quite sure i agree with that sentiment, analyzing tests gives us further confidence our tests are doing what they are supposed to, and [phpstan analyzes it's own tests](https://github.com/phpstan/phpstan-src/blob/1.9.x/build/phpstan.neon#L17)

Reply:
So just handle `$this->app` as if it can be `null`. I shown you that removing `null` on testbench-core would also cause phpstan analyse to fail.